test/gtest/uct/uct_test.cc

```diff
@@ -453,31 +461,9 @@ std::vector<const resource*> uct_test::enum_resources(const std::string& tl_name
 
         uct_worker_destroy(worker);
         ucs_async_context_destroy(async);
-
-        set_cm_resources(all_resources);
     }
 
-    return filter_resources(all_resources, tl_name);
-}
-
-void uct_test::generate_test_variant(int variant,
```

Comment:
why need to remove? How will you generate variants for UCT?

Reply:
not used anywhere, CM variants are components, not TLs like here

Reply:
it will be used in tag offload tests, can you avoid removing it?

Reply:
maybe add it together with usage? I also need to add variants but different way for CMs, so at some point we should merge 2 approaches or keep them independent. It's hard to decide now for dead code.

Reply:
ok